web/themes/contrib/civictheme/includes/views.inc

```diff
@@ -250,14 +255,150 @@ function _civictheme_preprocess_views__exposed_form__group_filter(array &$variab
   $variables['group_filter'] = TRUE;
 }
 
+/**
+ * Preprocess views selected filters from the query params.
+ */
+function _civictheme_preprocess_views_view__selected_filters(array &$variables): void {
+  /** @var \Drupal\views\ViewExecutable $view */
+  $view = $variables['view'];
+
+  // Selected filters does not currently support ajax.
+  if ($view->ajaxEnabled()) {
+    return;
+  }
+
+  $variables['selected_filters'] = _civictheme_preprocess_views_view__selected_filters_list($view);
+
+  if (!empty($variables['selected_filters'])) {
+    $variables['selected_filters_clear_link'] = [
+      'url' => Url::fromRoute('<current>')->toString(),
+      'text' => t('Clear all'),
+      'attributes' => 'aria-label="' . t('Clear all filters') . '"',
+    ];
+  }
+}
+
 /**
  * Preprocess for a search page.
  *
  * @SuppressWarnings(PHPMD.ElseExpression)
  */
 function _civictheme_preprocess_views_view__search_page(array &$variables): void {
+  /** @var \Drupal\views\ViewExecutable $view */
   $view = $variables['view'];
   if ($view->id() == 'civictheme_search' && $view->current_display == 'page_1') {
     $variables['vertical_spacing'] = 'top';
   }
 }
+
+/**
+ * Preprocess the selected filters list.
+ */
+function _civictheme_preprocess_views_view__selected_filters_list(ViewExecutable $view): array {
+  $filter_types = _civictheme_preprocess_views_view__selected_filters_get_filter_types($view);
+  $query_params = \Drupal::request()->query->all();
+  $current_values = array_filter($query_params, function ($value) {
+    return !empty($value);
+  });
+
+  // Only process keys for available filters.
+  $permitted_keys = array_keys($filter_types);
+  $selected_filters = [];
+  $base_url = Url::fromRoute('<current>')->toString();
```

Comment:
this should go into `_civictheme_preprocess_views_view__selected_filters_list_create_filter_url()` function